### ISSUE TITLE
The problem of the enhanced results while using different loss functions?

### ISSUE THREAD

**Issue by YoungJay0612** [facebookresearch/demucs#149]
Hello, When I train the demucs model with  l1-loss，the enhanced result is very well, and the speech sense is always  satisfactory 。But  when I change the loss function from l1-loss to l1-loss+0.1* multi-resolution STFT loss， I just changed the stft_loss in the configuration file config.yaml to be true in the configuration file。But the enhanced speech introduced some noise.This is the example file [https://github.com/YoungJay0612/demucs-enhanced-result](url). I have thought a lot and have no idea now. Hopefully to hear from you.
